P/Powsybl/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+using BinaryBuilder, Pkg
+
+name = "Powsybl"
+version = v"0.1.0"
+sources = [
+    GitSource("https://github.com/powsybl/powsybl.jl.git", "0a9e3110c366808e7ac23cfd9e19f51793807392"),
+    GitSource("https://github.com/powsybl/pypowsybl.git", "cd5fea41bbfb2897fd71a6e63b2d07a465055699")
+]
+
+julia_versions = [v"1.7", v"1.8", v"1.9", v"1.10"]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+
+# Get binary for powsybl-java, generated with GraalVm
+if [[ "${target}" == *-mingw* ]]; then
+    wget https://github.com/powsybl/pypowsybl/releases/download/v1.7.0/binaries-v1.7.0-windows.zip -O powsybl-java.zip
+fi
+if [[ "${target}" == *-linux-* ]]; then
+    wget https://github.com/powsybl/pypowsybl/releases/download/v1.7.0/binaries-v1.7.0-linux.zip -O powsybl-java.zip
+fi
+if [[ "${target}" == *-apple-* ]]; then
+    wget https://github.com/powsybl/pypowsybl/releases/download/v1.7.0/binaries-v1.7.0-darwin.zip -O powsybl-java.zip
+fi
+unzip powsybl-java.zip -d $prefix
```

Comment:
Why don't you add these as dependencies?  They can't be cached if you download them manually inside the script like this.

Reply:
Do you mean using this : https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.Dependency ?
From what I undestand it is limited to binaries wrapped in a jll package ?
While this is something we could probably do later, the way to retrieve binaries included in powsybl-java will probably change because this will evolve when this part is correctly isolated in its own repository and not retrieved from powsybl python bindings.

Reply:
No, sorry, I meant sources, not dependencies

Reply:
Ok I see that you can add archive dependencies through ArchiveSource, I can change that.